packages/shopping-cart/src/pages/Register.jsx

```diff
@@ -36,14 +36,14 @@ export default function Register(props) {
       }, 2000);
     } 
     else {
-      alert("Wrong Credentials");
+      alert("Passwords does not match");
     }
   }
 
   return(
     <div className="bg-light p-5 my-5 col-5 mx-auto border border-2">
       <h3 className="text-center mb-4">Create your account</h3>
-      <form className="" onSubmit={handleSubmit}>
+      <form className="needs-validation" onSubmit={handleSubmit}>
```

Comment:
are we validating the following cases?
- if password and confirm password are both empty?
- is there valid email provided
- if email is empy?
- if user name is empty?
- do we have some minimum length for password ?

Reply:
same concerns for the login form.

Reply:
every field is required, and a valid email is also added but there is no condition for minimum length since fields are required therefore automatically min length is 1